Formula/emscripten.rb

```diff
@@ -154,15 +175,30 @@ def install
   end
 
   def post_install
+    return if File.exist?("#{Dir.home}/.emscripten")
```

Comment:
`emcc --generate-config` will error if this file exists, so let's skip the `post_install` and just warn in `caveats` if this file exists.